extensions/wikia/TrackingOptIn/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
-    "@wikia/tracking-opt-in": "^1.0.32",
+    "@wikia/tracking-opt-in": "github:wikia/tracking-opt-in#675927bf6f472d086184ecb8f7ef2a3125d59f8e",
```

Comment:
Remember to update after https://github.com/Wikia/tracking-opt-in/pull/51 is released.